.github/CODEOWNERS

```diff
@@ -0,0 +1,238 @@
+/helmfile.d/values/hnc/ @elastisys/goto-security
+/helmfile.d/values/networkpolicies/ @elastisys/goto-security
+/helmfile.d/values/networkpolicy/ @elastisys/goto-security
+/helmfile.d/values/opensearch/ @elastisys/goto-logging-stack
+/helmfile.d/values/podsecuritypolicies/ @elastisys/goto-security
+/helmfile.d/values/rclone/ @elastisys/goto-security
+/helmfile.d/values/thanos/ @elastisys/goto-monitoring-stack
+/helmfile.d/values/trivy/ @elastisys/goto-security
+/helmfile.d/values/userCRDs/ @elastisys/goto-security
+/helmfile.d/values/autoscaling-monitoring.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/calico-accountant.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/calico-default-deny.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/cert-manager.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/cluster-admin-rbac.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/dev-namespaces.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/dex.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/file-copier.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/ingress-nginx-probe-ingress.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/ingress-nginx.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/kubeapi-metrics.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/kube-prometheus-stack-* @elastisys/goto-monitoring-stack
+/helmfile.d/values/kube-state-metrics-extra-resource-metrics.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/kured.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/letsencrypt.yaml.gotmpl @elastisys/goto-ingress-auth
+/helmfile.d/values/metrics-server.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/minio.yaml.gotmpl @elastisys/goto-block-storage
+/helmfile.d/values/node-local-dns.yaml.gotmpl @elastisys/goto-kubernetes
+/helmfile.d/values/openstack-monitoring.yaml.gotmpl @elastisys/goto-logging-stack
+/helmfile.d/values/prometheus-alerts-sc.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/prometheus-blackbox-exporter-* @elastisys/goto-monitoring-stack
+/helmfile.d/values/prometheus-opensearch-exporter.yaml.gotmpl @elastisys/goto-logging-stack
+/helmfile.d/values/prometheus-user-alerts-wc.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/s3-exporter.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/sc-servicemonitor.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/tekton.gotmpl @elastisys/goto-continuous-delivery
+/helmfile.d/values/user-alertmanager.yaml.gotmpl @elastisys/goto-monitoring-stack
+/helmfile.d/values/user-rbac.yaml.gotmpl @elastisys/goto-security
+/helmfile.d/values/velero-* @elastisys/goto-security
+/helmfile.d/values/wc-servicemonitor.yaml.gotmpl @elastisys/goto-monitoring-stack
+
+#
+# Skipped files in helmfile.d/values
+# admin-namespaces-sc.yaml.gotmpl
```

Comment:
Shouldn't these have owners? Like `goto-kubernetes` or `goto-security` maybe

Reply:
It would be good with owners here as well, but I was not sure which since these files "belong" to a lot of components. 
I think `goto-security` makse some sense, since this includes the PSA labels. Not sure about `goto-kubernetes`, what was your reasoning for that?

Reply:
I agree that it's used by many different areas. But I'm just thinking since this is basically only creating standard namespaces, the issues is around namespaces and the ones that have the best knowledge about them should be kubernetes gotos.

Reply:
Ok, I see your point now. Let's go with kubernetes goto for now.